README.md

```diff
@@ -136,3 +136,38 @@ openstack esi node volume attach
 - `--port <port>`:  Neutron port (name or UUID)
 - `node`: Node (name or UUID)
 - `volume`: Volume (name or UUID)
+
+## `openstack esi switch <command>`
+
+These commands allow you to treat ESI as a switch.
+
+### `openstack esi switch vlan list`
+
+List VLANs and associated switch ports.
+
+```
+openstack esi switch vlan list
+```
+### `openstack esi switch port enable access`
+
+Attach VLAN to a switchport.
+
+```
+openstack esi switch port enable access
+   <switchport>
+   <vlan>
+```
```

Comment:
A general question about the command line: How do we specify the switch whose switchport we want to operate on? Also this switchport is the real switchport, right? like `Ethernet1/3`?

Reply:
When I say "whose switchport" I mean we will generally have more than one switch. So `Ethernet1/3` will be on a switch e.g. `cisco-1` and `cisco-2`.

Reply:
Ah, I'm assuming that each ESI only deals with a single switch right now - is that likely to change?

Switchport is the real switchport, correct!

Reply:
> When I say "whose switchport" I mean we will generally have more than one switch. So `Ethernet1/3` will be on a switch e.g. `cisco-1` and `cisco-2`.

Oh - it would be whichever switch ESI is configured to work with.

Reply:
> I'm assuming that each ESI only deals with a single switch right now - is that likely to change?

When we add port information to a baremetal node it does take in all the switch information (switch IP etc), so I assumed that for a separate node we could put a different switch. I was under the assumption that multiple switches are supported out of the box. Sorry if I missed this somewhere.

Reply:
> > I'm assuming that each ESI only deals with a single switch right now - is that likely to change?
> 
> When we add port information to a baremetal node it does take in all the switch information (switch IP etc), so I assumed that for a separate node we could put a different switch. I was under the assumption that multiple switches are supported out of the box. Sorry if I missed this somewhere.

Oh, no worries! The important specification is actually a switch identifier, which corresponds to a configuration in ml2_conf.ini. We could in theory have multiple switch configurations in ml2_conf.ini, and different nodes could specify different switches. Is that likely? I had thought that would actually be different instances of ESI.

Reply:
> Is that likely?

Yes, in fact, it is extremely unlikely that an ESI cluster will only manage a single switch. The expectation is that it will be used to manage multiple switches (only the switchports where the nodes connect to, not the interswitch links) and those switches may also be of different kind.

Reply:
Ah, okay! I'll add a required switch parameter.

Reply:
@naved001 okay, I've updated the commands - let me know what you think!